documents/store-operations/fulfillment/fulfillment-app-update.md

```diff
@@ -0,0 +1,32 @@
+# HotWax Commerce Fulfillment App: Migration Guide and User Manual
+
+## Introduction
+
+HotWax Commerce has enhanced its `Fulfillment` app with major UI improvements and new case-handling features. These updates address the challenges of managing kit products and streamline the order rejection process, making it easier for retailers and store associates to handle orders efficiently and accurately. 
+
+## Kit Products Visibility
+
+The new user interface improves handling of orders with kit products. The enhanced app provides clear identification and management of kit products within the order view. 
+
+### Identifying Kit Products:
+- **Kit Labels:** Kit items now display a `Kit Label` next to their identifiers in the order view. This visual cue helps store associates quickly recognize items that are part of a kit.
+- **Options Icon:** Click the `Options Icon` next to the kit label to view all items included in the kit. This feature is particularly useful for store associates when a customer orders multiple kit products in one order, enabling them to easily identify which items belong to each kit.
+
+
+### Managing Kit Products:
+- **Stock Visibility:** Stock visibility for the entire kit is based on the item with the least inventory. This ensures that store associates have an accurate view of the availability of kit items.
+- **Order Rejection:** Use the `Bin` icon to reject kit items. The selected rejection reason will be applied to all items in the kit, ensuring consistent handling of kit products.
+
+## Order Rejection Options
+
+Previously, store associates had to manually select rejection reasons for each order item, leading to unnecessary order splits. The updated Fulfillment App streamlines this process by allowing retailers to decide between partial rejection and and full shipment rejection.
```

Comment:
We should explain, what do we mean by partial and full shipment rejection. How can these use cases be different.

Reply:
@Amir-HC